instrumentation/opentelemetry-instrumentation-asgi/tests/test_asgi_middleware.py

```diff
@@ -368,7 +372,7 @@ def validate_outputs(
                 "name": "GET / http send",
                 "kind": trace_api.SpanKind.INTERNAL,
                 "attributes": {
-                    SpanAttributes.HTTP_STATUS_CODE: 200,
```

Comment:
The task is to substitute the use of SpanAttributes with other modules inside `opentelemetry.semconv`, not to change what is asserted here. If we don't have a replacement with the same value then they should stay untouched